dotnet/src/Connectors/Connectors.AzureAIInference/Services/AzureAIInferenceChatCompletionService.cs

```diff
@@ -16,9 +17,11 @@ namespace Microsoft.SemanticKernel.Connectors.AzureAIInference;
 /// <summary>
 /// Chat completion service for Azure AI Inference.
 /// </summary>
+[Obsolete("Dedicated AzureAIInferenceChatCompletionService is deprecated. Use OllamaApiClient.AsChatCompletionService() instead.")]
```

Comment:
Should it be like this?
```suggestion
[Obsolete("Dedicated AzureAIInferenceChatCompletionService is deprecated. Use ChatCompletionsClient.AsChatClient().AsChatCompletionService() instead.")]
```